assets/app/widgets/suse-lazy-spinner/suse-lazy-spinner.directive.spec.js

```diff
@@ -0,0 +1,125 @@
+/*global bard $httpBackend assert $rootScope $compile $timeout */
+describe('SUSE Lazy Spinner Directive', function () {
+
+    var directiveElem,
+        scope;
+
+    beforeEach(function () {
+        bard.appModule('crowbarApp');
+
+        bard.inject('$rootScope', '$compile', '$httpBackend', '$timeout');
+
+        //Mock requests that are expected to be made
+        $httpBackend.expectGET('app/features/upgrade/i18n/en.json').respond({});
```

Comment:
From this directive's perspective we don't, but it's tested in the app context and the language file is actually fetched somewhere in the background.